### PR TITLE
clean(test): remove unused and document running unit test

### PR DESCRIPTION
**Pull request by corinnekrych**

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @corinnekrych for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin.

**Comment by ppitonak** [comment]
FYI @ljelinkova

File: README.md [hunk 1]
```diff
@@ -31,9 +31,37 @@ Requires `node` version 8.3.0 and `npm` 5.3.0. Consider using [Node Version Mana
...
-
 Run `npm start`. This will start the UI with live reload enabled. Then navigate to <http://localhost:3000>.
 
+### Run test
+
+We use [jest](https://github.com/facebook/jest) test loader because it's faster than karma execution.
+
+#### All tests
+```
+npm run test
```
Comment (joshuawilson) [inline README.md]: `test` is a keyword in npm and therefore we don't need the `run` command. `npm test` works and is shorter.

Comment (christianvogt) [reply README.md]: Hah. I always still type `npm run test`

**Pull request merged**